platzky/db/json_db.py

```
@@ -1,46 +1,57 @@
-# TODO make all language specific methods to be available without language
-# this will allow to have a default language and if there is one language
-# there will be no need to pass it to the method or in db
-
-
+# TODO: Make all language-specific methods available without language parameter.
+# This will allow a default language and if there is one language,
+# there will be no need to pass it to the method or in db.
 class Json(DB):
+    """In-memory JSON database implementation."""
+
     def __init__(self, data: dict[str, Any]):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)